### PR TITLE
use project name as docker image name

### PR DESCRIPTION
**Pull request by mittet0**
Please take a look together with https://github.com/clippings/service-core-template/pull/7

### CODE REVIEW

File: src/main/resources/docker.gradle [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
+    to { image = getDockerImage() }
     container {
         environment(
                 SPRING_PROFILES_ACTIVE: "default",
                 DT_DB_SCHEMA: "public",
                 DT_DB_URL: "jdbc:postgresql://localhost:5432/postgres?currentSchema=\${DT_DB_SCHEMA}",
                 DT_DB_USERNAME: "postgres",
-                DT_DB_PASSWORD: "postgres"
+                DT_DB_PASSWORD: "postgres",
+                SPRING_APPLCIATION_NAME: "${projectName}".toString()
```
Comment (pecataToshev) [inline src/main/resources/docker.gradle]: why do we need `.toString()`

Comment (mittet0) [reply src/main/resources/docker.gradle]: if not using toString() - this is the result 
[GStringImpl cannot be cast to java.lang.String](https://stackoverflow.com/questions/29079574/gstringimpl-cannot-be-cast-to-java-lang-string)

Comment (vvalchev) [reply src/main/resources/docker.gradle]: hmm, then could it be
```
                SPRING_APPLCIATION_NAME: project.name.toLowerCase(),
```

Comment (vvalchev) [reply src/main/resources/docker.gradle]: oh, there is a typo in `SPRING_APPLCIATION_NAME'

Comment (mittet0) [reply src/main/resources/docker.gradle]: Comments applied 
type in spring application name fixed 
replace usage of toString with project.name.toLowerCase()

**Review by vvalchev — APPROVED** [review]

File: src/main/resources/docker.gradle [hunk 2]
```diff
@@ -1,15 +1,28 @@
 apply plugin: 'com.google.cloud.tools.jib'
 
 
+ext.projectName = project.name.toLowerCase()
```
Comment (vvalchev) [inline src/main/resources/docker.gradle]: we don't need this as a global variable. Maybe it should be inside the function below, or you can even quote it inside the string:
```
"${registry}${project.name.toLowerCase()}:${version}"
```

Comment (mittet0) [reply src/main/resources/docker.gradle]: done

**Review by pecataToshev — APPROVED** [review]

**Pull request merged**